corehq/pillows/mappings/user_mapping.py

```diff
@@ -85,6 +85,23 @@
                 'last_name': {'type': 'string'},
                 'password': {'type': 'string'},
                 'registering_device_id': {'type': 'string'},
+                'reporting_metadata': {'dynamic': False,
+                                       'properties': {
+                                           'last_submissions': {
+                                               'dynamic': False,
+                                               'properties': {
+                                                    'submission_date': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
```

Comment:
E126 continuation line over-indented for hanging indent
E501 line too long (387 > 115 characters)